### ISSUE TITLE
[web-client]: Add policy functions

### ISSUE THREAD

**Issue by onmax** [nimiq/core-rs-albatross#2834]
## New issue checklist

- [X] I have read all of the [`README`](https://github.com/nimiq-network/core/blob/master/README.md) 
- [X] I have searched [existing issues](https://github.com/nimiq-network/core/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

## Question or Feature Request

I would like to access from the Web Client the policy constants and functions as we currently have in the [RPC/policy](https://github.com/nimiq/core-rs-albatross/blob/albatross/rpc-interface/src/policy.rs).

Basically to have something like: `client.policy.get_macro_block_before(XXX)` or `client.policy.get_macro_block_of(XXX)`

I don't think that's possible with the current version, so my question is:

Does this make sense? Would it be possible?

PD: Asking this publicly so other people in the future can read what we decide on this matter

**Comment by sisou** [comment]
There should be a `Policy` class exported from the package, which has all those methods and constants.

**Comment by sisou** [comment]
https://www.nimiq.com/developers/build/web-client/reference/classes/Policy
